app/(tabs)/you.tsx

```diff
@@ -1,21 +1,42 @@
 import { router } from 'expo-router';
-import React from 'react';
+import React, { useContext } from 'react';
 import { useTranslation } from 'react-i18next';
 import { View } from 'react-native';
 
 import { CustomButton } from '@/components';
+import ProfileHeader from '@/components/modules/userdetails/ProfileHeader';
 import SafeView from '@/components/ui/box/SafeView';
-import useLogout from '@/hooks/auth/UseLogout';
+import { GlobalContext } from '@/context/GlobalContext';
 import { ButtonType } from '@/util/ButtonType';
 
 export default function You() {
   const { t } = useTranslation();
-  const { mutate: logout } = useLogout();
+  const { userData } = useContext(GlobalContext);
+
+  function getName() {
```

Comment:
could you extract that function and give it arguments firsName, lastname, and use switch ?